src/renderer_vk.cpp

```diff
@@ -6251,6 +6251,12 @@ VK_DESTROY
 		const VkImageLayout oldLayout = m_currentImageLayout;
 		const VkImageLayout oldSingleMsaaLayout = m_currentSingleMsaaImageLayout;
 
+        auto numLayers = _numLayers;
+        if(m_type == VK_IMAGE_VIEW_TYPE_CUBE || VK_IMAGE_VIEW_TYPE_CUBE_ARRAY)
```

Comment:
This `m_type == VK_IMAGE_VIEW_TYPE_CUBE || VK_IMAGE_VIEW_TYPE_CUBE_ARRAY` is kind of nonsensical code. You need to test `m_type` against enum.

I fixed it here: https://github.com/bkaradzic/bgfx/commit/cbdc8bf77c6585d91c8262c15a06e378cbad9b31

https://github.com/bkaradzic/bgfx/commit/cbdc8bf77c6585d91c8262c15a06e378cbad9b31